src/main/java/com/rocketden/main/game_object/Submission.java

```diff
@@ -0,0 +1,27 @@
+package com.rocketden.main.game_object;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+
+@Getter
+@Setter
+public class Submission {
+
+    private Player player;
+
+    private String code;
+
+    // The time that the submission was received.
+    private LocalDateTime startTime = LocalDateTime.now();
+
+    /**
+     * Fields to store the score result.
+     * TODO: Eventually, there may exist a new Result object.
```

Comment:
Hadn't thought about that - a Results object seems like a great idea

Reply:
Thanks! I agree, that sounds good for the future.